great_expectations/core/expectation_suite.py

```diff
@@ -215,6 +206,17 @@ def _has_been_saved(self) -> bool:
         key = self._store.get_key(suite=self)
         return self._store.has_key(key=key)
 
+    @property
+    def expectation_configurations(self) -> list[ExpectationConfiguration]:
+        return [exp.configuration for exp in self.expectations]
+
+    @expectation_configurations.setter
```

Comment:
i used a setter to give an informative error message - this can go away once the class is fully refactored and `expectation_configurations` are no longer used. I do notice that we're raising a `RuntimeError` here, and it should actually be an `AttributeError`, so will update that.